core/tests/protractor/embedding.js

```diff
@@ -172,22 +248,56 @@ describe('Embedding', function() {
 
     users.createUser('embedder2@example.com', 'Embedder2');
     users.login('embedder2@example.com', true);
-    adminPage.reloadExploration('protractor_test_1.yaml');
-
-    // Change language to Thai, which is not a supported site language.
-    general.openEditor('12');
-    editor.setLanguage('ภาษาไทย');
-    editor.saveChanges('Changing the language to a not supported one.');
-    // We expect the default language, English
-    checkPlaceholder('Type a number');
-
-    // Change language to Spanish, which is a supported site language.
-    general.openEditor('12');
-    editor.setLanguage('español');
-    editor.saveChanges('Changing the language to a supported one.');
-    checkPlaceholder('Ingresa un número');
-
-    users.logout();
-    general.checkForConsoleErrors([]);
+
+    // Create an exploration.
+    workflow.createExploration();
+    general.getExplorationIdFromEditor().then(function(expId) {
+      explorationId = expId;
+      
+      editor.setContent(forms.toRichText('Language Test'));
+      editor.setInteraction('NumericInput');
+      editor.addResponse('NumericInput', forms.toRichText('Nice!!')
+        , 'END', true, 'IsLessThanOrEqualTo', 0);
```

Comment:
The comma should go at the end of the previous line.

Better still, break the line after the '('. I.e.:

```
      editor.addResponse(
        'NumericInput', forms.toRichText('Nice!!'),
        'END', true, 'IsLessThanOrEqualTo', 0);
```